src/Rings/mpoly-graded.jl

```diff
@@ -367,6 +583,7 @@ function addeq!(a::MPolyElem_dec, b::MPolyElem_dec)
 end
 
 length(a::MPolyElem_dec) = length(a.f)
+total_degree(a::MPolyElem_dec) = total_degree(a.f)
```

Comment:
I am not an expert, but here the degree is not related to the grading? Just curiosity.

Reply:
I understand that the total_degree function returns the total degree of a polynomial with respect to the standard grading. The degree function returns the degree of a homogeneous polynomial with respect to the given grading. If this is confusing, let us reconsider ...

Reply:
No, it is fine with me. I am happy to keep it.